bcda/models/service.go

```diff
@@ -18,31 +33,29 @@ type Service interface {
 }
 
 type cclfBeneficiaryService interface {
-	// GetNewAndExistingBeneficiaries, when supplied with the "since" parameter, returns two arrays
-	// the first array contains all NEW beneficaries that were added to CCLF since the date supplied
-	// the second array contains all EXISTING benficiaries that have existed in CCLF since prior to the date supplied
-	GetNewAndExistingBeneficiaries(cmsID string, since time.Time) (newBeneficiaries, beneficiaries []*CCLFBeneficiary, err error)
-
-	// GetBeneficiaries retrieves all beneficiaries associated with the ACO, contained in one array
-	GetBeneficiaries(cmsID string) ([]*CCLFBeneficiary, error)
+	GetQueJobs(cmsID string, job *Job, resourceTypes []string, since time.Time, reqType RequestType) (queJobs []*que.Job, err error)
 }
 
 const (
 	cclf8FileNum = int(8)
 )
 
-func newService(r Repository, cutoffDuration time.Duration, lookbackDays int) Service {
-	serviceInstance = &service{
+func NewService(r Repository, cutoffDuration time.Duration, lookbackDays int) Service {
+	return &service{
 		repository:     r,
 		logger:         log.StandardLogger(),
 		cutoffDuration: cutoffDuration,
```

Comment:
This is the standard, non-runout cutoff duration.  Should we make it clear by changing the name?